sdk/core/switcher/entry.S

```diff
@@ -27,6 +27,21 @@
 #define SPILL_SLOT_pcc 24
 #define SPILL_SLOT_SIZE 32
 
+/*
+ * RISC-V custom-use mcause values for RTOS internal use.
+ * See also include/priv/riscv.h, which defines the same names and values in C
```

Comment:
maybe we should have a c header file with the macro definitions and include it in both places?

Reply:
That's a great idea, thanks.